.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: generic
```

Comment:
So what are the steps that travis does for our application from this file?

Reply:
Right now it install flutter and dependencies, then it runs all the tests and creates a coverage report, then it reads the coverage report and always passes. Finally it sends a notification to discord on the build status

Reply:
Awesome